src/main/kotlin/no/nav/syfo/behandler/api/person/PersonBehandlerApi.kt

```diff
@@ -36,30 +35,15 @@ fun Route.registerPersonBehandlerApi(
                     token = token,
                 )
 
-                val personBehandlerDTOList = mutableListOf<PersonBehandlerDTO>()
-                val fastlege = behandlerService.getAktivFastlegeBehandler(
+                val behandlere = behandlerService.getBehandlere(
                     personident = requestPersonident,
-                    systemRequest = true,
                     token = token,
```

Comment:
Tror ikke dette vil virke: koden vil tro at token her er et veileder-azure-token, mens det egentlig er et idporten-token (?). Så må antakelig lage ny parameter for systemRequest slik metoden som erstattes har.

Reply:
Ja, det tenkte jeg ikke på 🙀